.github/linters/.golangci.yml

```diff
@@ -0,0 +1,3 @@
+run:
+  skip-dirs:
+    - net/tcp
```

Comment:
Why do we exclude this?

Reply:
I was not able to fix the errors. If you have some time, give it a try:

https://github.com/bio-routing/bio-rd/runs/1187348144?check_suite_focus=true

Reply:
So the new CI system would not build certain parts of the project. 
Sorry, wont merge.

Reply:
It's just the linter that has been added. It did not exist before.